tests/evhtests/l7_evh_crd_test.go

```diff
@@ -80,6 +80,8 @@ func TestMain(m *testing.M) {
 	os.Setenv("POD_NAMESPACE", utils.AKO_DEFAULT_NS)
 	os.Setenv("SHARD_VS_SIZE", "LARGE")
 	os.Setenv("AUTO_L4_FQDN", "default")
+	os.Setenv("AKO_ID", "1")
+	lib.SetAKOID()
```

Comment:
Can we call SetAKOID with the AKO_ID as argument in stead of setting env variable ?